src/cloud-api-adaptor/podvm-mkosi/Makefile

```diff
@@ -36,8 +36,8 @@ define run_mkosi_in_container
 	docker run --rm --privileged \
 		-v "$(shell pwd)":/workspace \
 		-w /workspace \
-		fedora:40 \
-		bash -c "dnf install -y mkosi && mkosi --tools-tree=default --tools-tree-release=41 $(1)"
+		fedora:41 \
+		bash -c "dnf install -y bubblewrap coreutils git python3; mkdir -p /build; git clone -b v22 https://github.com/systemd/mkosi /build/mkosi; export PATH="/build/mkosi/bin:$$PATH"; mkosi --tools-tree=default --tools-tree-release=41 $(1)"
```

Comment:
```suggestion
		bash -c "dnf install -y bubblewrap coreutils git python3; mkdir -p /build; git clone -b $(MKOSI_VERSION) https://github.com/systemd/mkosi /build/mkosi; export PATH="/build/mkosi/bin:$$PATH"; mkosi --tools-tree=default --tools-tree-release=41 $(1)"
```

Reply:
Yeah sorry about that - I switched to hard-coded when I was debugging and forgot to change it back